src/library/analysisfeature.cpp

```diff
@@ -130,9 +150,14 @@ void AnalysisFeature::activate() {
 
 void AnalysisFeature::analyzeTracks(QList<TrackId> trackIds) {
     if (!m_pTrackAnalysisScheduler) {
+        const int numAnalyzerThreads = numberOfAnalyzerThreads();
+        kLogger.info()
+                << "Using"
```

Comment:
spaces so that the text looks ok

Reply:
Formatted with to auto formatter. Keeping the various string components on separate lines aligns them.